Gemfile.lock

```diff
@@ -208,9 +208,9 @@ GEM
     multi_xml (0.6.0)
     nenv (0.3.0)
     nio4r (2.5.8)
-    nokogiri (1.13.1-x86_64-darwin)
+    nokogiri (1.13.4-x86_64-darwin)
```

Comment:
Do we still need nokogiri? Or can we remove it?

Reply:
its removed from the gemfile but for some reason when i bundle install it stays in the .lock